vendor/modules.txt

```diff
@@ -2,7 +2,7 @@
 ## explicit; go 1.20
 github.com/NVIDIA/go-gpuallocator/gpuallocator
 github.com/NVIDIA/go-gpuallocator/internal/links
-# github.com/NVIDIA/go-nvlib v0.6.1
+# github.com/NVIDIA/go-nvlib v0.6.2-0.20240928162840-41955a08425b
```

Comment:
Do we want to cut a `go-nvlib` release for this change first?

Reply:
sure, how can I help?

Reply:
now points to `v0.7`